### DISCUSSION TITLE
B68843F5-4863-DA49-43AD-3376CAA93B85

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#14] [General]
# Milky Way Center

This addon adds to Celestia famous black holes and most real stars (catalogs E, N, B, GCIRS and partially S added) of the Milky Way center.
Some stars have real orbits, and some are still stable because their orbital period is unknown.
Some stars have their own characteristics (double star systems or accretion discs in Be stars)
The addon radially modifies black holes (especially Sagittarius A *), transforming them from something outdated and incomprehensible into really dangerous objects.
It is also worth noting that the add-ons as shown by various tests with such aggressive accretion discs work without brakes.

https://celestia.mobi/addon?item=B68843F5-4863-DA49-43AD-3376CAA93B85&lang=en

**Comment by Dirgham2015** [comment]
I think this is cool

**Comment by amonn809** [comment]
I think this is perfect, This is a reference to Sagittarius A.

**Comment by WasiAdiyatHasan** [comment]
This is the best black hole reference

**Comment by bird0cosmic** [comment]
ooo

**Comment by rutkiewiczenzo-afk** [comment]
I really love this

**Comment by den19k1052-creator** [comment]
Imba👍🏻👍🏻

**Comment by Cardenasdidier23-max** [comment]
Where is this at? Also this is nice.
![IMG_7724](https://github.com/user-attachments/assets/48c4ee31-605f-4e3b-8382-d3b40c6be5f9)

**Comment by LangLangFCS** [comment]
Wow!It's so cool.I like it.

**Comment by curnyloveautentik-maker** [comment]
The biggest step, but it forms like a black hole, if it can be found at home in the galaxy, like the Via Lacheta, we call them Ton-618 black hole
